TypingPro/main.py

```diff
@@ -80,3 +80,10 @@ def msgToScreen(screen, msg, positions):
     pygame.display.update()
         
 pygame.quit()
+# End
+# <-- LICENCE info -->
+'''
+TypingPro
+Copyright Ved Rathi
+Licensed under MIT (https://github.com/Ved-Programmer/TypingPro/blob/master/LICENSE)

```

Comment:
Show we include a link to the license or just include the license itself? I am thinking that including the actual MIT License inside the script could have a disadvantage of not being up to date. We would have to keep updating the license in the code if we ever want to change the license type of license year or modify the license for any reason. Plus if someone has downloaded the code and we change the license the later then the downloaded version of the code would have the outdated license. This is a disadvantage however, the advantage is that the license would be there in the code always, it doesn't matter if we delete this repository or something like that. The license would stay there forever. It all will be in one location. So, if someone wants to see the license then the person can just read it right in the code. I am kind of confused which solution would be better, what do you think, I would like your opinion on that. Should we include the license itself or just a link to the license.